### ISSUE TITLE
Chinese user signup issue - Google image verification is blocked

### ISSUE THREAD

**Issue by TimCliff** [steemit/condenser#515]
Many of the Chinese users are reporting that they cannot sign up using the new verification process, because the image verification uses Google, and Google is blocked for them.

**Comment by jcalfee** [comment]
Someone suggeted: http://www.yinxiangma.com/

**Comment by abitmore** [comment]
If we still have interest, here is an alternative to Google's reCaptcha in China: http://www.geetest.com/exp_normal , which is being used by some big sites.

Another alternative is AliYun's service: https://help.aliyun.com/document_detail/28322.html.

**Comment by roadscape** [comment]
Still a problem? We are using a different google captcha now

**Comment by abitmore** [comment]
Yes, it's still a problem.

```
Failed captcha verification, please try again
```

Currently in the community, Chinese people are advised that it's needed to use a secure VPN or proxy to be able to visit steemit.com.

By the way, in [this comment](https://github.com/steemit/condenser/issues/86#issuecomment-263067722) @sneak said:
> Working around censorship firewalls is not a design goal at this time. Perhaps in the future.

If it's still the case, I guess we can label this issue as `wontfix`.

**Comment by roadscape** [comment]
Thanks @abitmore

**Comment by sneak** [comment]
I would love to support the chinese market better in the future. Perhaps we can host our own captcha instead of using recaptcha.

**Comment by plink01001** [comment]
Can we disable the captcha for Chinese users until a more permanent solution can be found?

**Comment by gl2748** [comment]
Discussion from slack.
```
timcliff [1:30 AM] 
@sneak the current status of the steemit.signup process for Chinese users is they cannot create accounts due to the Google image verification: https://github.com/steemit/condenser/issues/515. last we talked about this, there were no immediate plans to fix. I know the team has a *lot* going on right now, so there is likely no time for it, but I wanted to bring it to your attention in case it was something that could be fit in seeing as we will likely be getting a lot of Chinese users interested in the platform in early Jan.
GitHub
Chinese user signup issue - Google image verification is blocked · Issue #515 · steemit/condenser
Many of the Chinese users are reporting that they cannot sign up using the new verification process, because the image verification uses Google, and Google is blocked for them.
 
 


sneak [7:46 PM] 
Ahh right


[7:46] 
Well, the issue with that is that if we fix it and we gain any traction then steemit.com will end up blocked at the GFW so it will have been wasted effort.


[7:47] 
I'm not opposed to disabling the captcha for Chinese IPs


[7:47] 
but it will only work temporarily


timcliff [10:56 PM] 
The Chinese community has actually been running a local condenser instance from Hong Kong or Taiwan, so if steemit.com gets blocked they still plan to live on. Their main thing is that they cannot handle signups, but after HF 20 they are planning to try adding support for that too.
 2 replies Last reply today at 10:00 AM View thread

timcliff [10:57 PM] 
If we can unblock the Chinese IPs from captcha for the January exchange launch, that will solve the immediate need
```

**Comment by plink01001** [comment]
Closing issue, as all signup functionality has been moved to the https://github.com/steemit/faucet repository.

@jnordberg is this issue resolved with the new faucet?

**Comment by jnordberg** [comment]
Yep
